### ISSUE TITLE
Grunt task implementation

### ISSUE THREAD

**Issue by DeChrish** [mocha-parallel/mocha-parallel-tests#193]
Can you provide the grunt implementation ?

**Comment by 1999** [comment]
@DeChrish I can't say that I know a lot of things about `grunt`. Can you provide a sample `grunt` task for `mocha`?

**Comment by DeChrish** [comment]
@1999 - https://www.npmjs.com/package/grunt-mocha

**Comment by 1999** [comment]
Nah, I totally get it. What I don't understand is the usual flow of the projects using Gunt. For me, Grunt is a task runner where you define a set of tasks like "gunt bundle" which bundles the front-end code etc. But what's the idea of using `mocha` in a Grunt task? Why not use it directly from the package.json scripts and run "npm test" instead?

**Comment by 1999** [comment]
@DeChrish ^
